src/tools/kata-ctl/src/check.rs

```diff
@@ -197,6 +206,101 @@ pub fn check_official_releases() -> Result<()> {
     Ok(())
 }
 
+#[cfg(any(target_arch = "x86_64"))]
+pub fn check_kernel_module_loaded(module: &KernelModule) -> Result<()> {
+    const MODPROBE_PARAMETERS: &str = "-n --first-time";
```

Comment:
That seems wrong, you are passing two arguments as a single `.arg()`.

```
> modprobe -n --first-time fuse
modprobe: ERROR: could not insert 'fuse': Module already in kernel
> modprobe '-n --first-time' fuse
modprobe: invalid option -- ' '
```

Reply:
@c3d thank you for your inputs, the arguments are now separated in two `.arg()` argument values.